tests/test_features.py

```diff
@@ -0,0 +1,13 @@
+from cyclic_boosting import CBPoissonRegressor
+import numpy as np
+
+
+def test_poisson_regressor_feature_importance(get_inputs):
```

Comment:
Wouldn't this fit better in integration tests?

Reply:
In a way, yes, but since this is testing particular functionality of `CyclicBoostingBase` (in particular, associated to `Feature`), I would keep it separate from the "end-to-end" tests. Plus, I would like to add more testing for `features.py` in the future. I am flexible, so if you want I can put this in `test_integration.py`

Reply:
I'm ok to keep it here.